src/commons/Sequence.cpp

```diff
@@ -300,11 +297,6 @@ void Sequence::mapProfile(const char * profileData, bool mapScores, unsigned int
             if(sumProb > 0.9){
                 MathUtil::NormalizeTo1(&profile[l * PROFILE_AA_SIZE], PROFILE_AA_SIZE);
             }
-            if(nullCnt==PROFILE_AA_SIZE) {
```

Comment:
I think this was used to mask a profile column. Is this still possible?

Reply:
The respective commit message:
"nullCnt was only used to reset a profile column to zero but that could
only happen, if the column contained only zeroes already. In that case,
sumProb will also be zero and the normalize function is not called."